MapboxNavigation/NavigationViewController.swift

```diff
@@ -266,6 +266,12 @@ public class NavigationViewController: UIViewController, RouteMapViewControllerD
         }
     }
     
+    @objc public var showsEndOfRoute: Bool = true {
```

Comment:
Can you add some documentation? It might be more descriptive to call this `showsEndOfRouteFeedbackUI` or something similar.